packages/clerk-js/src/ui/components/devPrompts/EnableOrganizationsPrompt/index.tsx

```diff
@@ -0,0 +1,247 @@
+import type { __internal_EnableOrganizationsPromptProps } from '@clerk/shared/types';
+import { css } from '@emotion/react';
+
+import { Modal } from '@/ui/elements/Modal';
+import { InternalThemeProvider } from '@/ui/styledSystem';
+
+import { Flex } from '../../../customizables';
+import { Portal } from '../../../elements/Portal';
+import { basePromptElementStyles, PromptContainer } from '../shared';
+
+const EnableOrganizationsPromptInternal = (props: __internal_EnableOrganizationsPromptProps) => {
+  const ctaText = 'componentName' in props ? `<${props.componentName} /> component` : props.utilityName;
+
+  return (
+    <Portal>
+      <Modal
+        canCloseModal={false}
+        contentSx={() => ({ animation: 'unset' })}
```

Comment:
```suggestion
        contentSx={() => ({ alignItems: 'center' })}
```

I think we can follow similar implementations to center the modal, and preserve the fade up animation.